neural_compressor/adaptor/torch_utils/weight_only.py

```diff
@@ -323,6 +372,8 @@ def awq_quantize(model, weight_config={}, absorb_dict={}, dataloader=None, n_sam
         mse_range (bool, optional):  whether enable clip for weight by checking mse. Defaults to True.
         calib_func: a custom inference function to replace dataloader and iters.
         n_blocks: split model into block number to avoid OOM.
+        return_int (bool, optional): Choose return fp32 or int32 model.
```

Comment:
People may not understand why INT32 model. How about reusing export function to save INT4 model (avoid many functionalities inside **quantize**)?